README.md

```diff
@@ -493,6 +493,75 @@ if __name__ == "__main__":
         logging.exception(exc)
         exit(2)
 ```
+
+## Table Schemas in ComponentBase
+
+In cases of a static schemas of output/input tables, the schemas can be defined using a JSON Table Schema.
+For output mapping these json schemas can be automatically turned into out table definitions.
+
+### JSON Table Schema example file
+
+```json
+{
+  "name": "product",
+  "description": "this table holds data on products",
+  "parent_tables": [],
+  "primary_keys": [
+    "id"
+  ],
+  "fields": [
```

Comment:
any particular reason to use different naming to the Keboola Connection API https://keboola.docs.apiary.io/#reference/tables/create-table-definition/create-new-table-definition ?

Reply:
Good point. Speaking of which, we are using a name TableDefinition for I/O mapping representation in this library also, so it may be a bit confusing too. But we came up with it years before this API was existing.